ui/src/shared/actions/sources.js

```diff
@@ -105,8 +103,7 @@ export const deleteKapacitorAsync = kapacitor => async dispatch => {
   } catch (err) {
     dispatch(
       publishNotification(
-        'error',
-        'Internal Server Error. Could not delete Kapacitor config.'
+        internalServerErrorNotifications().deleteKapacitorFail
```

Comment:
this is funky and would be cleaned up by the requested syntax above that using `sprintf`